### ISSUE TITLE
Port issues

### ISSUE THREAD

**Issue by flakjacket95** [php-opencloud/openstack#109]
The retrieve and update functions for the Port class do not properly update the local object after calling the API.

**Comment by haphan** [comment]
@flakjacket95 can you provide code that can reproduce the bug?

**Comment by flakjacket95** [comment]
Yes, essentially the issue is that once the API completes its request, it does not repopulate the object locally. In the source code, there are lines that populate the fields.
`$this->populateFromResponse($response);`
That line is missing in the Port objects methods for update and retrieve. With an active connection to an OpenStack server the following will print out an empty object that contains only the Port ID.
  ```php
$networking = $openstack->networkingV2();
$port = $networking->getPort({id});
$port->retrieve();
print_r($port);
```
I've fixed this issue in #110 
Screenshots of this behavior are attached.
Before the modification:
![before](https://cloud.githubusercontent.com/assets/10423137/24590770/373d024a-17c1-11e7-9b1a-cd8bc36db06d.png)
After the modification:
![after](https://cloud.githubusercontent.com/assets/10423137/24590771/373e6d10-17c1-11e7-8cfc-350743d48973.png)

**Comment by haphan** [comment]
Fixed in #110
